docs/Release_3.6.0.md

```
@@ -0,0 +1,91 @@
+# ThoughtEcho v3.6.0 发布说明 / Release Notes
+> 📖用户指南/User Guide: https://note.shangjinyun.cn/user-guide.html
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)